### ISSUE TITLE
Okta Verify via SMS Phone Number Label

### ISSUE THREAD

**Issue by JonathanDCrowe** [okta/okta-signin-widget#3570]
### Describe the bug

Our accessibility team identified an issue with the phone number field that is displayed when you selected the option to be texted a link to setup Okta Verify.  Specifically, the phone number textbox has no accessible name, likely because the aria-label attribute is an empty string.

![image](https://github.com/okta/okta-signin-widget/assets/8659545/d808c4a5-7aeb-4862-a9d9-1a8aa573bbf0)

### Reproduction Steps

In the security method enrollment screen select "Okta Verify". Click "Can't Scan". Select the option to have a linked texted to you. The user is then presented with the option to enter their phone number, which is where this accessibility issue occurs.

### SDK Versions

7.16.1
